libs/mrwebrtc-unityplugin/src/native_renderer.cpp

```diff
@@ -39,12 +40,19 @@ void I420VideoFrame::CopyFrame(const mrsI420AVideoFrame& frame) {
   size_t ysize = (size_t)ystride * height;
   size_t usize = (size_t)ustride * height / 2;
```

Comment:
Should we use `(height + 1) / 2` to round up and avoid truncating chroma for the last line here?

Reply:
Not sure, it seems to have been working like this without issue?

Reply:
I just tried adding the height+1, but noticed at the top of the frame it adds a 1 pixel horizontal line of distorted pixels on 1080p. I assume that means it is correct how it currently is.

Reply:
That sounds more like a double bug to me. If the height is odd, the last Y line still needs UV chroma. As is, we basically only have luminance for that last line. Sounds more to me like changing to `(height + 1) / 2` uncovered another bug somewhere else.

Reply:
Or maybe the line of pixels at the top is correct and the issue is in my stream? It's just copying the I420AVideoFrame struct directly.

Your saying it should be like this, correct?
```
  size_t ysize = (size_t)ystride * height;
  size_t usize = (size_t)ustride * (height + 1) / 2;
  size_t vsize = (size_t)vstride * (height + 1) / 2;
```

Reply:
So I did some more comparisons on some different streams. The difference may actually be fairly negligible, see attached photo. So if +1 is more technically correct I think that is fine. I have committed it with height + 1.
![compare](https://user-images.githubusercontent.com/53834195/110699900-30e70c00-81a4-11eb-8914-9f82f4a3b5e6.png)